src/pudl/output/ferc1.py

```diff
@@ -1155,39 +1155,52 @@ class OffByFactoid(NamedTuple):
 @asset
 def _out_ferc1__explosion_tags(table_dimensions_ferc1) -> pd.DataFrame:
     """Grab the stored tables of tags and add inferred dimension."""
-    # Also, these tags may not be applicable to all exploded tables, but
-    # we need to pass in a dataframe with the right structure to all of the exploders,
-    # so we're just re-using this one for the moment.
-    rate_base_tags = _rate_base_tags(table_dimensions_ferc1=table_dimensions_ferc1)
+    rate_tags = _get_tags("xbrl_factoid_rate_base_tags.csv", table_dimensions_ferc1)
```

Comment:
i decided to standardize the tags bc now there are a million so its either `_get_tags` which just grabs a csv and fills in the dimensions or the per-existing `_aggregatable_dimension_tags` which is now used for all three of our dimensions.

then instead of a bunch of little merges i used concat to squish em all together.